analyze.sh

```diff
@@ -15,7 +15,7 @@ if [ -d "$ARTIFACT_DIR" ]; then
 fi
 
 MAX_BYTES=3879731 # ~3.7 MiB
-VENDORS_MAIN_BYTES=$(du -b `find public/dist -type f -name 'vendors~main-chunk*js'` | cut -f1)
+VENDORS_MAIN_BYTES=$(du -b `find public/dist -type f -name 'vendors~*-bundle-*js'` | cut -f1 | sort -n | tail -1)
```

Comment:
not sure if this is the best way to do this.. i spent some time messing with the webpack configuration, but couldn't get webpack to include "main chunk" in the the biggest vendor file, so this was the next best option